bootstrapoptions_features.go

```diff
@@ -0,0 +1,72 @@
+package europi
+
+import (
+	"errors"
+	"time"
+)
+
+const (
+	DefaultMainLoopInterval time.Duration = time.Millisecond * 100
```

Comment:
Indeed - I totally agree. 100 ms was a 'safe" default value decision made with very little consideration with regards to user experience on my point when I made it. In my day job and when working with UI in video games, this value is largely there to prevent code from running away and stealing all the frame budget, but not long enough to make players upset if it manages to make it out to production.

When it comes to EuroPi functionality within the Go bootstrap paradigm, it's intended that there will be at least 2 goroutines running simultaneously, the main loop (where CV/VOct is calculated and set) and the UI loop (where user inputs from knobs and buttons are handled and passed over to the main loop). This poses a bit of a problem, though, where a small understanding of multi-threading is required and care to avoid resource contention becomes more important. To combat this, I've made most of the EuroPi resources automatically avoid contention - though there is nothing that speaks directly about this to the consumers of the SDK.

I'll add some documentation on how to avoid resource contention between UI and main 'threads' and additionally come up with some documentation about how to tune the main and UI loops timers with points about tuning and caveats.